pkg/validate/raw/validate/source_spec_validator.go

```diff
@@ -193,31 +193,23 @@ func HelmSpec(helm *v1beta1.HelmBase, rs client.Object) status.Error {
 	return nil
 }
 
-// CheckValuesFileRefs validates that the ConfigMaps specified in the RSync ValuesFileRefs exist and have the
-// specified data key.
-func CheckValuesFileRefs(ctx context.Context, cl client.Client, valuesFileRefs []v1beta1.ValuesFileRefs, rs client.Object) status.Error {
+// CheckValuesFileRefs checks that the ConfigMaps specified by valuesFileRefs exist, are immutable, and have the provided data key.
+func CheckValuesFileRefs(ctx context.Context, cl client.Client, rs client.Object, valuesFileRefs []v1beta1.ValuesFileRef) status.Error {
```

Comment:
with validate in the package name, "check" is redundant here

Reply:
fixed